helper/testhelpers/consul/consulhelper.go

```diff
@@ -26,7 +26,7 @@ func (c *Config) APIConfig() *consulapi.Config {
 // the Consul version used will be given by the environment variable
 // CONSUL_DOCKER_VERSION, or if that's empty, whatever we've hardcoded as the
 // the latest Consul version.
-func PrepareTestContainer(t *testing.T, version string) (func(), *Config) {
+func PrepareTestContainer(t *testing.T, version string, isEnterprise bool) (func(), *Config) {
```

Comment:
Nit: Can we add [t.Helper](https://pkg.go.dev/testing#T.Helper) call in here so that the line error emits from unit tests where this gets call if we ever error out of this function? 

```suggestion
func PrepareTestContainer(t *testing.T, version string, isEnterprise bool) (func(), *Config) {
	t.Helper()
```

Reply:
Added